tests/test_equilibrium.py

```diff
@@ -61,6 +64,9 @@ def test_map_coordinates():
     )
     assert not np.any(np.isnan(out))
 
+    iota = np.broadcast_to(iota, shape=(n,))
+    np.testing.assert_allclose(eq.map_clebsch_coords(coords, iota), out)
```

Comment:
The new method doesn't match the old one, so I must be doing something wrong.

Reply:
Removing `eq.change_resolution` fixes this. I think this is because changing radial resolution affects `map_coordinates` but it does not effect `map_clebsch_coords`, or some inconsistency like that.